### ISSUE TITLE
Change `blog/list.html` to use `.Title` instead of `.LinkTitle`

### ISSUE THREAD

**Issue by at055612** [google/docsy#838]
Noticed that `blog/list.html` is using `.LinkTitle` for the blog entry links in the main page.
```html
<h5 class="mt-0 mb-1"><a href="{{ .RelPermalink }}">{{ .LinkTitle }}</a></h5>
```

This is inconsistent with `docs/list.html` (`partials/section-index.html`)
```html
<h5>
    <a href="{{ $manualLink }}"{{ with .Params.manualLinkTitle }} title="{{ . }}"{{ end }}{{ with .Params.manualLinkTarget }} target="{{ . }}"{{ if eq . "_blank" }} rel="noopener"{{ end }}{{ end }}>{{- .Title -}}</a>
</h5>

```

I think `blog/list.html` should be changed to use `.Title` as the `.LinkTitle` will typically be shorter to fit in the left hand nav pane but in the main pane there is more space for the full title. i.e. 

```html
<h5 class="mt-0 mb-1"><a href="{{ .RelPermalink }}">{{ .Title }}</a></h5>
```

As an example, on the docsy example site some of the blog entries have different titles to link titles, i.e. "Announcing Docsy" vs "Easy documentation with Docsy"

![image](https://user-images.githubusercontent.com/22818309/149797694-bdc70963-d434-40b4-b51d-3b5d4df1b2cf.png)

![image](https://user-images.githubusercontent.com/22818309/149797828-17e3b55d-2ffa-49f9-9124-ff6b1cd2786d.png)

Happy to raise a PR if people agree with this.

**Comment by NaincyKumariKnoldus** [comment]
Hey @at055612 Can I raise a PR for this please?

**Comment by at055612** [comment]
@NaincyKumariKnoldus go for it.

**Comment by NaincyKumariKnoldus** [comment]
Sure @at055612

**Comment by NaincyKumariKnoldus** [comment]
Hey @at055612 I have raised a PR for this! Review this please.

**Comment by at055612** [comment]
PR #839 looks fine to me but it is not my decision to make.  The question is whether the community want to see the LinkTitle or the Title in the main pain.

By the way if you raise a PR against an issue, link back to the issue in the PR, e.g. by putting this in the PR description.
```
Fixes #839
```

**Comment by NaincyKumariKnoldus** [comment]
okay @at055612

**Comment by chalin** [comment]
Thanks for catching that @at055612, and for the PR @NaincyKumariKnoldus. I agree with the proposed change, and have approved the PR.

**Comment by NaincyKumariKnoldus** [comment]
Thanks @chalin

**Comment by chalin** [comment]
Closed by #839
